app/javascript/react/screens/App/Overview/screens/PlanWizard/PlanWizardBody.js

```diff
@@ -69,10 +35,12 @@ PlanWizardBody.propTypes = {
   plansBody: PropTypes.object,
   planWizardGeneralStep: PropTypes.object,
   planWizardVMStep: PropTypes.object,
+  transformationMappings: PropTypes.array,
   hideAlertAction: PropTypes.func
 };
 
 PlanWizardBody.defaultProps = {
+  wizardSteps: [],
```

Comment:
If on 29 yah say this is array of objects and default value as an empty array (with no objects) 🤔

Reply:
An array with zero objects is still an array of objects :) I think maybe this one just shouldn't have a default prop though... it would never be useful.